### DISCUSSION TITLE
Semptec ZX7007/7008 infrared wifi heaters

### DISCUSSION THREAD

**Discussion by kimhartlev** [make-all/tuya-local#1824] [General]
Hello,

I have added support for Semptec ZX7007/8 infrared wifi heater in the attached yaml file. Feel free to add it to the repository.

Cheers,
Kim

<img width="679" alt="Screenshot 2024-04-12 at 22 31 35" src="https://github.com/make-all/tuya-local/assets/13569619/6ac1cde8-512d-4d86-b360-0d5c2e338614">

[zx7007_infrared_heater.yaml.zip](https://github.com/make-all/tuya-local/files/14963350/zx7007_infrared_heater.yaml.zip)

**Comment by make-all** [comment]
Please file a new device request, and provide all the details the template asks for if you want to request a new device. Discussions are not tracked the way issues are.

**Reply by kimhartlev** [reply]
I don't want to file a new request, I just wanted to share the device I created so others could benefit.
